src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -2502,7 +2500,7 @@ internal void Run(bool inputLoopIsNested)
                                 // Evaluate any suggestions
                                 if (previousResponseWasEmpty == false)
                                 {
-                                    EvaluateSuggestions(ui);
+                                    EvaluateFeedbacks(ui);
```

Comment:
Ideally, the processing of feedback should not be tied to a specific host.  One of the ideas was to have "suggestions" as part of the ErrorRecord and present it as part of formatting.